runtime/gc_modron_startup/arrayCopy.cpp

```diff
@@ -274,7 +274,7 @@ initializeReferenceArrayCopyTable(J9ReferenceArrayCopyTable *table)
 	table->forwardReferenceArrayCopyWithCheckIndex[j9gc_modron_wrtbar_cardmark] = forwardReferenceArrayCopyWithCheckAndAlwaysWrtbarIndex;
 	table->forwardReferenceArrayCopyWithCheckIndex[j9gc_modron_wrtbar_cardmark_incremental] = forwardReferenceArrayCopyWithCheckAndAlwaysWrtbarIndex;
 	table->forwardReferenceArrayCopyWithCheckIndex[j9gc_modron_wrtbar_cardmark_and_oldcheck] = forwardReferenceArrayCopyWithCheckAndAlwaysWrtbarIndex;
-	table->forwardReferenceArrayCopyWithCheckIndex[j9gc_modron_wrtbar_realtime] = forwardReferenceArrayCopyWithCheckAndAlwaysWrtbarIndex;
+	table->forwardReferenceArrayCopyWithCheckIndex[j9gc_modron_wrtbar_satb] = forwardReferenceArrayCopyWithCheckAndAlwaysWrtbarIndex;
```

Comment:
this table should be expanded for satb and oldcheck too (fine for future changes)